experiment/run_experiment.py

```diff
@@ -593,6 +666,14 @@ def main():
         parser.error('The sum of runners and measurers cpus is greater than the'
                      ' available cpu cores (%d)' % os.cpu_count())
 
+    if args.random_seed_corpus:
+        if args.no_seeds:
+            parser.error(
+                'You cannot start an experiment with no_seeds option if'
```

Comment:
Maybe the message could be "Cannot enable options 'random_seed_corpus'  and 'no_seeds' at the same time".

Reply:
fixed